raster/src/main/scala/geotrellis/raster/viewshed/R2Viewshed.scala

```diff
@@ -90,8 +90,12 @@ object R2Viewshed extends Serializable {
     * @param  cols  The number of columns
     * @param  rows  The number of rows
     */
-  def generateEmptyViewshedTile(cols: Int, rows: Int) =
-    ArrayTile.empty(IntConstantNoDataCellType, cols, rows)
+  def generateEmptyViewshedTile(cols: Int, rows: Int, debug: Int = 0) = {
```

Comment:
It looks like `debug` should be `Boolean` here, if you still want to keep this flag.